terraform/environments/data-and-insights-wepi/secrets.tf

```diff
@@ -1,3 +1,5 @@
+#### This file can be used to store secrets specific to the member account ####
```

Comment:
when there are already resources in secrets, I tend to remove the info heading, as the teams already are using it and therefore do not need informing, but I guess it is just a preference